include/_pokey.h

```diff
@@ -73,6 +73,7 @@ struct __pokey_read {
     unsigned char   skstat; /* serial port status */
 };
 
+
```

Comment:
spurious whitespace change?

Reply:
> spurious whitespace change?

It's a style change that brings the file closer to the way that Uz formatted his files.